src/core/inlines.js

```diff
@@ -121,14 +126,27 @@ function inlineLinkMatches(matched) {
     .split("/", 2)
     .map(s => s.trim());
   const [isFor, content] = parts.length === 2 ? parts : ["", parts[0]];
-  return hyperHTML`<a data-link-for="${isFor}" data-xref-for="${isFor}">${content}</a>`;
+  const processedContent = processInlineContent(content);
+  return hyperHTML`<a data-link-for="${isFor}" data-xref-for="${isFor}">${processedContent}</a>`;
 }
 
 function inlineCodeMatches(matched) {
   const clean = matched.slice(1, -1); // Chop ` and `
   return hyperHTML`<code>${clean}</code>`;
 }
 
+function processInlineContent(text) {
+  if (inlineCodeRegExp.test(text)) {
+    // We use a capture group to split, so we can process all the parts.
+    return text.split(/(`[^`]+`)(?!`)/).map(part => {
+      return part.startsWith("`")
+        ? inlineCodeMatches(part)
+        : processInlineContent(part);
+    });
+  }
+  return hyperHTML`${text}`;
```

Comment:
Maybe just `text`?

Reply:
Just text confuses hyper, in that it then coerces all other items in the array to text.

Reply:
Uhm, what array do you mean?

Reply:
Ah okay, this is a recursive function.

Reply:
But maybe `document.createTextNode(text)`? I think we prefered it on other parts of the code.

Reply:
good suggestion.